docs/search/extensibility/solr_document_field_mappers.md

```diff
@@ -48,20 +48,15 @@ Each mapper implements two methods, by the same name, but accepting different ar
 Mappers can be used on the extension points by registering them with the [service container](php_api.md#service-container) by using service tags, as follows:
 
 - All block documents
-    - `ContentFieldMapper`
-    - `ibexa.solr.field_mapper.block`
+    - `ibexa.search.solr.field.mapper.block`
 - All block documents per translation
-    - `ContentTranslationFieldMapper`
     - `ibexa.solr.field_mapper.block_translation`
```

Comment:
```suggestion
    - `ibexa.search.solr.field.mapper.block.translation`
```

Reply:
yeah. my bad. I looked by mistake at service id, not service tag in this case